blogs/julyblog.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "ECH Institute's July celebration"
+date: 2025-08-01
+author: Team ECH
+image: https://x.com/ECHInstitute/header_photo
+---
+# ECH Institute's July celebration 
```

Comment:
This can be deleted. I forgot this line isn't needed and just leads to a weird duplicate line in the page. The `title` in the header section is used for the big heading.

<img width="1002" height="660" alt="Screenshot 2025-09-29 at 3 49 25 PM" src="https://github.com/user-attachments/assets/91603394-655b-4d07-82e0-5b769bf76c28" />